src/react/components/pages/predict/uploadToTrainingSetView.tsx

```diff
@@ -0,0 +1,85 @@
+import { Customizer, ICustomizations, Label, Modal, PrimaryButton, Spinner, Stack, SpinnerSize } from '@fluentui/react';
+import React from 'react';
+import { getDarkGreyTheme, getPrimaryGreenTheme, getPrimaryGreyTheme } from '../../../../common/themes';
+import './uploadToTrainingSetView.scss';
+import { strings } from "../../../../common/strings";
+interface IUploadToTrainingSetViewProp {
+    onConfirm?: () => Promise<void>;
+}
+interface IUploadToTrainingSetViewState {
+    hideModal: boolean;
+    isLoading: boolean;
+}
+export class UploadToTrainingSetView extends React.Component<IUploadToTrainingSetViewProp, IUploadToTrainingSetViewState>{
+    constructor(props) {
+        super(props);
+        this.state = {
+            hideModal: true,
+            isLoading: false,
+        };
+        this.open = this.open.bind(this);
+        this.close = this.close.bind(this);
+        this.onConfirm = this.onConfirm.bind(this);
+    }
+    open() {
+        this.setState({
+            hideModal: false,
+            isLoading: false
+        });
+    }
+    close() {
+        this.setState({ hideModal: true });
+    }
+    async onConfirm() {
+        this.setState({ isLoading: true });
+        if (this.props.onConfirm) {
+            await this.props.onConfirm();
+        }
+        this.close();
+    }
+    render() {
+        const dark: ICustomizations = {
+            settings: {
+                theme: getDarkGreyTheme(),
```

Comment:
you need to apply getDefaultDarkTheme() - otherwise the spinner looks like white circle (not progress visible)
```suggestion
                theme: getDefaultDarkTheme(),
```

<img width="534" alt="Screen Shot 2020-08-27 at 9 26 52 AM" src="https://user-images.githubusercontent.com/64093224/91470510-a5caa800-e849-11ea-81b7-cd7a6d789ba3.png">

Reply:
fixed